### PR TITLE
Ingestion: ALTA 2023

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

File: data/xml/2023.alta.xml [hunk 1]
```diff
@@ -0,0 +1,258 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2023.alta">
+  <volume id="1" ingest-date="2024-03-08" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of The 21st Annual Workshop of the Australasian Language Technology Association</booktitle>
```
Comment (mjpost) [inline data/xml/2023.alta.xml:5]: The → the

**Review by mjpost — COMMENTED** [review]
Please fix "the The" in 2019–2022 ALTA book titles, and adjust the ingestion script / aclpub checker to look for (a) "the the" (after lowercasing) and (b) "The" in the middle of a booktitle.

File: data/xml/2023.alta.xml (hunk 1 above)
Comment (anthology-assist) [reply data/xml/2023.alta.xml]: Done.

**Review by mjpost — APPROVED** [review]
It would be great to also fix the ingestion script to look for "the the" but I'll leave it up to you.

**Comment by anthology-assist** [comment]
> It would be great to also fix the ingestion script to look for "the the" but I'll leave it up to you.

Created https://github.com/acl-org/acl-anthology/issues/3139

**Pull request merged**